scripts/app.js

```diff
@@ -0,0 +1,105 @@
+const getBooks = () => {
```

Comment:
- [ ] For this part of the project I should be able to see a default list of Books, kindly add some books to your list 😄

Reply:
This issue has been addressed but I don't think it was necessary for this milestone.